src/astro/databases/google/bigquery.py

```diff
@@ -28,11 +47,9 @@ class BigqueryDatabase(BaseDatabase):
     logic in other parts of our code-base.
     """
 
-    NATIVE_PATHS = {FileLocation.GS: "gs_to_bigquery"}
-    NATIVE_PATHS_SUPPORTED_FILE_TYPES = {
-        FileType.CSV: "CSV",
-        FileType.NDJSON: "NEWLINE_DELIMITED_JSON",
-        FileType.PARQUET: "PARQUET",
+    NATIVE_PATHS = {
+        FileLocation.GS: "gs_to_bigquery",
```

Comment:
What's the value (`gs_to_bigquery`) in this dict used for?

Reply:
@kaxil they are the function names that are supposed to be called for a certain path(source, destination). So now the mapping is - 
```
NATIVE_PATHS = {
        FileLocation.GS: "load_gs_file_to_bigquery",
        FileLocation.S3: "load_s3_file_to_bigquery",
    }
```
and below code is used to run a specific function.
https://github.com/astronomer/astro-sdk/blob/32b6f157a0cf7a64a150cc246a5ac70d4fc39410/src/astro/databases/google/bigquery.py#L164